american-physiological-society.csl

```diff
@@ -212,11 +212,15 @@
           </choose>
         </else-if>
         <else>
-          <text variable="container-title" suffix=" " form="short" font-style="italic"/>
-          <text variable="volume"/>
-          <text variable="publisher" suffix="."/>
-          <text variable="page" prefix=": " suffix=", "/>
-          <text macro="date" prefix=" " suffix="."/>
+          <group delimiter=" ">
```

Comment:
could we wrap these two groups in another group with delimiter=": " and suffix="."? My concern is that you'd not get the "." if there are neither date nor pages (or is that too hypothetical?)